### PR TITLE
Flat binaries

### PR DESCRIPTION
**Pull request by emetsger**
Fix for #8

### CODE REVIEW

**Comment by birkland** [comment]
I get test failures.  It looks like the test is referring to a binary file that isn't actually present in the test package?

`
Tests run: 4, Failures: 0, Errors: 4, Skipped: 0, Time elapsed: 0.21 sec <<< FAILURE! - in org.dataconservancy.packaging
.impl.PackageFileAnalyzerTest
testBinaryResourceNoContainer(org.dataconservancy.packaging.impl.PackageFileAnalyzerTest)  Time elapsed: 0.163 sec  <<<
ERROR!
java.lang.RuntimeException: Error processing non-container binary resources: /mvn/dcs-package-ingest/package-ingest-impl
/target/test-classes/test_pkg/data/bin/curl/NoParent.txt (No such file or directory)
        at java.io.FileInputStream.open0(Native Method)
        at java.io.FileInputStream.open(FileInputStream.java:195)
        at java.io.FileInputStream.<init>(FileInputStream.java:138)
        at org.dataconservancy.packaging.impl.PackageFileAnalyzer.populateFileResource(PackageFileAnalyzer.java:189)
        at org.dataconservancy.packaging.impl.PackageFileAnalyzer.lambda$getContainerRoots$1(PackageFileAnalyzer.java:11
5)
        at java.util.Iterator.forEachRemaining(Iterator.java:116)
        at org.dataconservancy.packaging.impl.PackageFileAnalyzer.getContainerRoots(PackageFileAnalyzer.java:113)
        at org.dataconservancy.packaging.impl.PackageFileAnalyzerTest.testBinaryResourceNoContainer(PackageFileAnalyzerT
est.java:149)
`

**Comment by emetsger** [comment]
Fixed.  The directory containing the file was ignored, and apparently must be removed from .gitignore for GitHub to process a commit that overrides the .gitignore.

File: package-ingest-impl/src/test/java/org/dataconservancy/packaging/impl/PackageFileAnalyzerTest.java [hunk 1]
```diff
@@ -26,8 +25,11 @@
 import java.net.URISyntaxException;
 import java.net.URL;
 import java.util.Collection;
+import java.util.function.Function;
```
Comment (birkland) [inline package-ingest-impl/src/test/java/org/dataconservancy/packaging/impl/PackageFileAnalyzerTest.java]: These imports are newly added, but not used

**Comment by emetsger** [comment]
@birkland ok cleaned up the lint!  I'm happy to merge it if you're cool with it

**Pull request closed without merging**